web/pdf_page_view.js

```diff
@@ -483,33 +454,123 @@ var PDFPageView = (function PDFPageViewClosure() {
           pageNumber: self.id,
           cssTransform: false,
         });
+      };
 
-        if (!error) {
-          resolveRenderPromise(undefined);
-        } else {
-          rejectRenderPromise(error);
+      var paintTask = this.renderer === CANVAS_RENDERER ?
+        this.paintOnCanvas(canvasWrapper) :
+        this.paintOnSvg(canvasWrapper);
```

Comment:
Nit: Given that the default is `canvas`, to me it seems a bit more logical to invert this condition:
```javascript
var paintTask = this.renderer === SVG_RENDERER ?
  this.paintOnSvg(canvasWrapper) :
  this.paintOnCanvas(canvasWrapper) :
```